eclipsePlugin/src/de/tobject/findbugs/decorators/ResourceBugCountDecorator.java

```diff
@@ -53,7 +325,8 @@ public String decorateText(String text, Object element) {
             }
             return text;
         }
-        return decorateText(text, item.getMarkerCount(false));
+        int markerCount = getMarkerCount(item);
```

Comment:
Why extract this local variable?

Reply:
Why not. But I will inline that in next commit.